.github/workflows/python-workflow.yaml

```diff
@@ -0,0 +1,66 @@
+# This workflow is based on the Github documentation workflow template.
+# See: https://github.com/actions/starter-workflows/blob/main/ci/python-package.yml
+
+name: python-build
+
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches: 
+      - main
+  schedule:
+    # This job runs every Monday at 9am PST.
+    - cron: "0 16 * * 1"
+
+jobs:
+  test-with-dremio:
+    runs-on: ubuntu-latest
+    defaults:
+        run:
+          working-directory: python
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
```

Comment:
Why limited to these 4? Can we instead put in a lower bound only?

Reply:
3.9 is not compatible with PyArrow, please see comment in the updated PR.